### ISSUE TITLE
UnicodeEncodeError

### ISSUE THREAD

**Issue by superqwe** [gsuitedevs/python-samples#57]
## Expected Behavior
Working quickstart

## Actual Behavior
Traceback when run quikstart.py

```
Getting the upcoming 10 events
2018-12-20 Traceback (most recent call last):
  File "D:\[omitted]\quickstart.py", line 39, in <module>
    main()
  File "D:\[omitted]\quickstart.py", line 36, in main
    print(start, event['summary'])
  File "D:\Env\myenv\lib\encodings\cp850.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_map)[0]
UnicodeEncodeError: 'charmap' codec can't encode character '\U0001f393' in position 0: character maps to <undefined>
```

## Steps to Reproduce the Problem

1. python quickstart.py

## Specifications

- Python 3.5.3
- Windows 10

**Comment by erickoledadevrel** [comment]
Sorry for the slow reply. Can you clarify for which quickstart (API) was this?

**Comment by sqrrrl** [comment]
Looks like calendar from the stack trace (event) and failing because the event summary contains a unicode character (graduation cap emoji.)

**Comment by sqrrrl** [comment]
More info at https://stackoverflow.com/questions/30539882/whats-the-deal-with-python-3-4-unicode-different-languages-and-windows/30551552#30551552

TL;DR - Python versions prior to 3.6 can't write unicode to the windows console without some additional hints. See the link for details on work arounds.

Quick start should run fine w/unicode in events on windows w/python 3.6+ without modification.

Suggest closing w/o action.
